crm_phonecall/__manifest__.py

```diff
@@ -1,11 +1,10 @@
-# Copyright 2017 Tecnativa - Vicent Cubells
-# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+# For more information contact me on mireda2012@gmail.com
 
 {
     "name": "CRM Phone Calls",
-    "version": "13.0.1.2.2",
+    "version": "14.0.1.0.0",
     "category": "Customer Relationship Management",
-    "author": "Odoo S.A., Tecnativa, Odoo Community Association (OCA)",
+    "author": "Odoo Community Association (OCA)",
```

Comment:
Same here

Reply:
git was showing problem with "no Odoo Community Association (OCA)" author found in manifest.
i'll fix it